notebooks/rosa-demo-open-ai-wfaq.ipynb

```diff
@@ -0,0 +1,997 @@
+{
```

Comment:
> I am not sure if this can be added to the case above. https://python.langchain.com/en/latest/modules/memory/types/buffer_window.html.
> 
> As quoted: "`ConversationBufferWindowMemory` keeps a list of the interactions of the conversation over time. It only uses the last K interactions. This can be useful for keeping a sliding window of the most recent interactions, so the buffer does not get too large"
> 
> If not here, we can keep this option for future exploration.
> 
> Otherwise, \lgtm
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/#comment-1218280457)_

That's a good point, this will lead to a chatbot like solution. I think we should explore it in the coming PRs. 

@codificat If there are no other suggestions, could we merge this?

Reply:
Agreed that we can explore more of memory and other improvements in future PRs.

As this one has been in the oven for quite a while now, it feels like we should merge it and move forward.

I took the liberty of editing the `notebooks/README.md` to add an entry for the notebook (new commit added).

Let's merge?

Reply:
@suppathak that's a fair point, let's add the chatbot like workflow in the coming PRs. @codificat Thanks for adding it to the Readme 💯